src/Spreadsheet/ColisActivDataExporter.php

```diff
@@ -27,7 +27,11 @@ public function __construct(EntityManagerInterface $entityManager)
         $this->entityManager = $entityManager;
     }
 
-    public function export(\DateTime $start, \DateTime $end): string
+    public function export(
+        \DateTime $start,
+        \DateTime $end,
+        bool $includeAllTasks = false
```

Comment:
This does not belong to the `DataExporterInterface` interface. 

Instead, you should move this to the constructor, and create another service with the `ColisActivDataExporter` class.